src/directives/uiSref.ts

```diff
@@ -87,7 +87,7 @@ export class UISref implements OnChanges {
    * <a uiSref="hoome">Home</a>
    * ```
    */
-  @Input('uiSref') state: string;
+  @Input('uiSref') state: StateOrName;
```

Comment:
Should this be changed on line 125 too?

Reply:
I'm confused? L125 in this file?

https://github.com/ui-router/angular/blob/a34d95a3994bc207aa3365b1c666486f72ec748c/src/directives/uiSref.ts#L125

Reply:
oops, typo.  I meant Line 135

```
  /** @internal */
  set uiSref(val: string) { // here
    this.state = val;
    this.update();
  }
```

Reply:
I do believe the setter is how the data-binding is applied 
```
  @Input('uiSref') state: StateOrName;
```

Reply:
Hmm well the decorator should bind the directive input "uiSref" to the class property "state". But nevertheless, whatever it's use, the setter should have the same type as the value it sets, I've updated it.